assignments/lambda-classes.js

```diff
@@ -1 +1,175 @@
-// CODE here for your Lambda Classes
+// For anyone reading my comments, I now feel like I'm cooking with fire.
+// class constructors are fantastic.
+const WEB17modules = ['Responsive Design I','Responsive Design II','Preprocessing I','Preprocessing II','JavaScript I','JavaScript II','JavaScript III','JavaScript IV']
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.location = attributes.location;
+        this.age = attributes.age;
+        this.gender = attributes.gender;
+    }
+    phrase() {
+        return `Hello, my name is ${this.name} and I'm from ${this.location}.`;
+    }
+}
+
+class Instructor extends Person {
+    constructor(childAttributes) {
```

Comment:
```suggestion
    constructor(props) {
```
To make your life easier, you can pretty much just always use props as the parameter for constructor/super. This will reflect the patterns of React, too.